kivymd/theming.py

```diff
@@ -19,6 +19,42 @@
 
 The main application class inherited from the `MDApp` class has the :attr:`theme_cls`
 attribute, with which you control the material properties of your application.
+
+Changing the theme colors
+-------------------------
+
+The standard theme_cls is designed to provide the standard themes and colors as
+defined by Material Design.
+
+We do not recommend that you change this.
+
+However, if you do need to change the standard colors, for instance to meet branding
+guidelines, you can do this by overloading the `color_definitions.py` object.
+
+* Create a custom copy of `color_definitions.py`,
+* Change the color codes in `colors` object in your custum `color_definitions.py`
+file as required. Note that you *cannot* change the names of the colors, you can only
+change the values. That is say, for instance, you cannot change the name of `Blue` and
+you cannot add `NavyBlue` but you change the value of the color blue to meet your 
+requirements.
+* Import the custom colors into your module and do something like this (where the custom 
+version of `color_defintions.py` is called `custom_palette.py`):
+
+.. code-block:: python
+
+    from custom_palette import colors
+
+    class Example(MDApp):
+        icons = list(md_icons.keys())[15:30]
+
+        def __init__(self, **kwargs):
+            super(Example, self).__init__(**kwargs)
+            self.theme_cls.colors = colors
+
```

Comment:
The documentation and example of this code do not explain anything to the user (and me too).

Reply:
Please do not take this as being awkward but I do not understand what it is that you do not understand?

Could you explain what is NOT explained?

... If you create a custom copy of color_definitions.py then all you need to do to reflect the changed colors in your app is add the line:

```
self.theme_cls.colors = colors
```

There is nothing else to do so please tell me what is missing rather than just saying that "it explains nothing"

Reply:
@runette If I don't understand how to change the palette, then other people will not understand either. Your example doesn't explain anything.

Reply:
@runette I also find it a bad decision to create a copy of the `color_definitions.py`